packages/reactivity/src/effect.ts

```diff
@@ -155,7 +155,7 @@ export function effect<T = any>(
     extend(_effect, options)
```

Comment:
This would lead to the `extend` call here to do a `for...in` loop on `EMPTY_OBJ`

Reply:
Sorry, I didn't get that

how about `options: ReactiveEffectOptions = {}`
to avoid`!options ||`